aggregate/load_aggregated.py

```diff
@@ -69,3 +69,37 @@ def initialize_dataframe_geo_index(geography, columns=[]):
     rv = pd.DataFrame(index=indicies[geography], columns=columns)
     rv.index.rename(geography, inplace=True)
     return rv
+
+
+"""this is specifically to use for housing security and quality March 4th POP data"""
+def load_clean_pop_data(ind_name_str: str) -> pd.DataFrame:
```

Comment:
another comment here: the `ind_name_str` parameter should be renamed to have "regex" in the same, something like `ind_name_regex`.